astrosat_users/models/models_customers.py

```diff
@@ -188,3 +188,23 @@ def invite(self, **kwargs):
 
         self.invitation_date = timezone.now()
         self.save()
+
+    def uninvite(self, **kwargs):
+
+        adapter = kwargs.get("adapter", get_adapter())
+        context = kwargs.get("context", {})
+        template_prefix = kwargs.get("template_prefix", "astrosat_users/email/uninvitation")
+
+        user = self.user
+        customer = self.customer
+        context.update({
+            "user": user,
+            "customer": customer,
+        })
```

Comment:
Previous uses of `context` I've seen have been like:

```python
context["user"] = user
context["customer"] = customer
```
Is this just a different way of doing the same thing? `update` suggests you are updating the user that already exists in the `context`, why not just overrwrite it?

Reply:
They do the same thing.  I don't think one is any better than another.  In general, if I am merging two dictionaries I will use `update` otherwise I will use explicit keys as you suggest.

Reply:
Your suggestion is a good one - using `[]` is much more efficient than using `update`.  I have done this in 86d2de2ab8072c6e996211a1a3947d0e068c5671.